interface/new/new_comprehensive_save.php

```diff
@@ -28,11 +28,9 @@
 // other interfaces can still read the data during this lock, however
 // sqlStatement("lock tables patient_data read");
 
-$result = sqlQuery("SELECT MAX(pid)+1 AS pid FROM patient_data");
-
+$result = sqlQuery("SELECT id AS pid FROM patient_data ORDER BY id DESC LIMIT 1");
```

Comment:
Help me out with the unrelated files. I don't see them when I do my commit. Even in the first commit, the dialog only showed me one file.

POST git-receive-pack (612 bytes)
Pushing to https://github.com/juggernautsei/openemr.git
To https://github.com/juggernautsei/openemr.git
   180cb6e..78ef24d  max-record-bug -> max-record-bug
updating local tracking ref 'refs/remotes/origin/max-record-bug'

Just trying to figure out how more files are slipping by without me seeing them in the dialog box.